clinica/iotools/utils/data_handling_cli.py

```diff
@@ -135,7 +135,14 @@ def define_options(self):
             "--ignore_scan_files",
             default=False,
             action="store_true",
-            help="If given, the session and scan files will not be read and added to "
+            help="If given, the scan files will not be read and added to "
+            "the final output. This may accelerate the procedure.",
+        )
+        iotools_options.add_argument(
+            "--ignore_session_scan_files",
+            default=False,
+            action="store_true",
+            help="If given, the session ans scan files will not be read and added to "
```

Comment:
There is a small typo in the help message, I think it should read: 
"If given, the session files will not be read and added to"

Reply:
Done in 74d55a9